src/sdk/namespace_fs.js

```diff
@@ -485,6 +627,10 @@ class NamespaceFS {
                     }
                     for (let i = marker_index; i < sorted_entries.length; ++i) {
                         const ent = sorted_entries[i];
+                        if (list_versions && marker_curr) {
+                            const ent_name = ent.name.substring(0, ent.name.indexOf('_mtime-'));
```

Comment:
here you can call get_file_name() instead

Reply:
sure